data.json

```
@@ -1876,15 +1876,6 @@
             ],
             "description": "A dynamic reflective pure object-oriented language supporting live programming inspired by Smalltalk."
         },
-        {
-            "name": "OpenFoodFacts-iOS",
-            "link": "https://github.com/openfoodfacts/openfoodfacts-ios",
-            "label": "help-wanted",
-            "technologies": [
-                "Swift"
-            ],
-            "description": "Collaborative, free and open database of food products from around the world. Scan barcode to get info or add a product"
-        },
         {
```

```suggestion
        {
            "name": "OpenFoodFacts",
            "link": "https://github.com/openfoodfacts/smooth-app",
            "label": "good first issue",
            "technologies": [
                "Dart"
            ],
            "description": "Collaborative, free and open database of food products from around the world. Scan barcode to get info or add a product"
        },
        {
```

Adding the new repository might look something like this.